config/holes.toml

```diff
@@ -2581,19 +2633,34 @@ preamble = '''
 </div>
 
 <p>
-    Each argument generally contains a prefix and a unit, either base or derived,
-    with the exception of symbols representing decimal multiples and sub-multiples
-    of kilogram because those are formed by prepending the letter "g" with a prefix
-    symbol. For example: "10^-6 kg" and "10^-9 kg" are the representations of "mg"
-    and "μg", respectively.
+    Each argument is an SI unit with a possible prefix. Print it as a product of a
+    power of ten and its base units. For example, <code>GW</code> (gigawatt) becomes
+    <code>10^9 kg m^2 s^-3</code>. Keep in mind the following exceptions:
 
-<p>
-    Convert each argument and print its respective prefix factor notation, joined
-    by the derivatives equivalent expressed in terms of base units. Derived units
-    evaluating to "1" should be omitted from the output. Additionally, print "10^1"
-    as "10" and "10^0" as "1". Notations that don't require alternative formatting
-    should resemble their default output so "MJ" becomes "10^6 kg m^2 s^-2", for
-    example.
+<ul>
+    <li>
+        The kilogram is a base unit with an inherent prefix. Other prefixes attach
+        to <b>g</b> (gram), and you should print their values relative to the
+        kilogram: <code>g</code> is <code>10^-3 kg</code> and <code>Mg</code>
+        is <code>10^3 kg</code>.
+    <li>
+        Instead of <code>10^1</code>, print <code>10</code>.
+    <li>
+        Instead of <code>10^0</code>, print <code>1</code>.
+    <li>
+        For <b>rad</b> and <b>sr</b>, only print a power of ten. (Example: for
+        <code>Mrad</b>, print <code>10^6</code>, not <code>10^6 1</code>.)
+    <li>
+        There is no fixed order among the base units. For <b>C</b>, you must
+        output <code>A s</code> rather than <code>s A</code>. We follow the
+        conventions of [NIST](https://www.nist.gov/pml/special-publication-330/sp-330-section-2#2.3.4).
```

Comment:
Does that syntax linkify? I think it ought to be a HTML <a>

Reply:
Oops haha, I will change it :)